### ISSUE TITLE
Browser source(?) causes PC to freeze up using 27.2.0

### ISSUE THREAD

**Issue by digiAlchem** [obsproject/obs-studio#5974]
### Operating System Info

Windows 10

### OBS Studio Version

27.2.0

### OBS Studio Log URL

https://obsproject.com/logs/v526BYQTRc1SVpUC

### Expected Behavior

When switching to a scene within OBS that has a browser source, and the browser source emits a Javascript error, OBS should handle it gracefully.

### Current Behavior

PC froze up, OBS stopped logging after Javascript error.

### Steps to Reproduce

With a browser source enabled, force a Javascript error (in this instance, `Uncaught ReferenceError` from an undefined variable being called) and switch to the scene containing the browser source.

Freezing occured three times over the course of two hours of streaming, sometimes instantly upon the scene switching and sometimes after 5 to 10 seconds.

### Anything else we should know?

Unable to upload anything but the latest log file, which is after I fixed the JS errors in the browser source (and stopped the issue from occuring) but these are the last lines of a log when the freezing did occur:
```
18:33:42.393: warning: Could not update timestamps for skipped samples.
18:33:42.411: warning: Could not update timestamps for skipped samples.
18:33:42.440: warning: Could not update timestamps for skipped samples.
18:33:42.678: adding 106 milliseconds of audio buffering, total audio buffering is now 149 milliseconds (source: Twitchmas Start Screen)
18:33:42.678: 
18:34:12.415: Max audio buffering reached!
18:34:12.415: adding 810 milliseconds of audio buffering, total audio buffering is now 960 milliseconds (source: FFXIV - Starting)
18:34:12.415: 
18:34:12.436: Source FFXIV - Starting audio is lagging (over by 67.57 ms) at max audio buffering. Restarting source audio.
18:34:25.102: obs-browser: Uncaught ReferenceError: currentSubGoalEnd is not defined (source: about:srcdoc:226)
18:34:25.107: obs-browser: Uncaught ReferenceError: currentSubGoalEnd is not defined (source: about:srcdoc:226)
18:34:25.109: obs-browser: Uncaught ReferenceError: currentSubGoalEnd is not defined (source: about:srcdoc:226)
18:34:25.142: obs-browser: Uncaught ReferenceError: currentSubGoalEnd is not defined (source: about:srcdoc:226)
```

PC froze at this point, with only audio from Spotify and the game (Final Fantasy XIV) playing - unable to Alt + Tab between windows, no other response from the OS, forcing a hard reboot of PC. The `currentSubGoalEnd` JS error was emitted from the browser source displaying a StreamElements overlay, and in previous versions of OBS (27.1.3 specifically) this failed gracefully and went unnoticed.

**Comment by Fenrirthviti** [comment]
This is something we've been tracking, but haven't posted a public issue about as we've been working around the clock to isolate and identify root cause.

We suspect there is a deadlock happening in DXGI on NVIDIA-based systems that is being triggered by our updated Browser Source (CEF version), but as it's happening very much outside of OBS itself, it has been difficult to track down exactly what is happening or why. We are engaged with NVIDIA on debugging this issue.

**Comment by astra-io** [comment]
Is this issue the same as using windowed projector and having it freeze and cause a GPU memory leak? 
Have to force quit OBS for performance to be restored.

**Comment by Fenrirthviti** [comment]
> Is this issue the same as using windowed projector and having it freeze and cause a GPU memory leak? Have to force quit OBS for performance to be restored.

No, this is a complete system freeze, not an OBS-specific freeze. I have not heard anything about that kind of issue, but sounds like it should be reported through the support channels for validation.

**Comment by RytoEX** [comment]
This should be fixed by OBS Studio 27.2.1.
